### ISSUE TITLE
Scenario Outline with examples in tables

### ISSUE THREAD

**Issue by MatthewSwarbrick** [bencompton/jest-cucumber#4]
```
Scenario Outline: Example scenario
      Given I have the following example data
        | Data1      | Data2      | Data3      |
        | <Data1> | <Data2> | <Data3> |
      When I do some action
      Then I can assert the result

  Examples:
      | Data1             | Data2             | Data3             |  
      | SampleData1 | SampleData2 | SampleData3 |
```

Instead of receiving "SampleData1" in my table data I get '\<Data1>\'. If I use the examples inline with steps like:

`Given I have the data field '<Data1>'`

Then I'll get the correct "SampleData1" coming through.

Am I able to use tables with scenario outlines or is this functionality not supported yet?

Thanks for your time

**Comment by bencompton** [comment]
If I understand correctly, you're expecting values in the table argument in your given step to be replaced with the values from your examples? This behavior is not implemented in this library, and as far as I know this behavior is also not implemented in Cucumber. Feel free to correct me if I'm overlooking something.

In my tests, your example above works exactly as I would expect. Your given step passes in the table argument as an array of objects into the step definition. If I then reference a value from the example table in the step text in the feature file as you mentioned with `Given I have the following example data "<Data1>"` and also update the step definition regex to contain be `/I have the following example data \"(.*)\"/ `, the step definition then gets two arguments, with the first being the value from the example table, and the second being the table argument for the given step.

**Comment by MatthewSwarbrick** [comment]
I've used these kind of behaviours using frameworks like Specflow brining cucumber/gherkin to c#

I've forked this repository and have what I would like working. I can do a pull request if you like? 

I guess I'd expect that in a Scenario Outline I can use data from my Examples table in the step text and in a step table.

**Comment by bencompton** [comment]
Fixed in #5
